### PR TITLE
Process the entries in the local config file

### PR DESCRIPTION
**Pull request by acinader**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -310,6 +311,11 @@ export class Config {
       }
     }
 
+    // Merge in local configs
+    if (typeof tree[local] == 'object') {
```
Comment (steve8708) [inline src/config.ts]: ===

Comment (acinader) [inline src/config.ts]: did you make a comment?

Comment (steve8708) [inline src/config.ts]: ah yes but I forgot just typing '===' into markdown renders a horizontal line, correct comment is `===` (as oppose to using `==`) :)

**Comment by acinader** [comment]
@steve8708  back at you

**Pull request merged**